src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/FormMetadataLoaderInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Metadata\FormMetadata;
+
+use Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerInterface;
+
+interface FormMetadataLoaderInterface extends CacheWarmerInterface
```

Comment:
I think this interface should not extend the `CacheWarmerInterface`... We've said that it should also be possible to have loaders that are not cached at all, right?